lava/lava-job-definitions/testplans/psa-poky.yaml

```diff
@@ -0,0 +1,58 @@
+{% extends "shared/templates/base.yaml" %}
+
+{% set job_name = "PSA Crypto"%}
+{% set lxc_creation = false %}
+{% set root_partition = 1 %}
+{% set login_prompt = "imx8mmevk login:" %}
+{% set prompts = [ "root@imx8mmevk:~#" ] %}
+
+{% block testplan %}
+- test:
+    timeout:
+      minutes: 20
+    namespace: target
+    definitions:
+    - from: inline
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: psa-arch-crypto-tests
+          description: psa-arch-crypto-tests
+        run:
+          steps:
+
+            - psa-arch-crypto-tests > /tmp/results
```

Comment:
Can we encapsulate all of this in a script?

Reply:
It could be in a script but I chose an inline definition as that doesn't require any git clones.

Reply:
Well, what's the difference with other tests we are running? Is it technically impossible to do the git clone?

Reply:
changed